recipes/cuda-compat/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+# Install to conda style directories
+[[ -d lib64 ]] && mv lib64 lib
+mkdir -p ${PREFIX}/cuda-compat
+
+cp -v lib/libcuda.so.${DRV_VERSION} ${PREFIX}/cuda-compat
+ln -sv libcuda.so.${DRV_VERSION} ${PREFIX}/cuda-compat/libcuda.so.1
+ln -sv libcuda.so.1 ${PREFIX}/cuda-compat/libcuda.so
+cp -v lib/libnvidia-nvvm.so.${DRV_VERSION} ${PREFIX}/cuda-compat
+ln -sv libnvidia-nvvm.so.${DRV_VERSION} ${PREFIX}/cuda-compat/libnvidia-nvvm.so.4
```

Comment:
How will we know when to change these soversions like ".so.4" and ".so.1"? Are those fixed or do they change?

Reply:
Looking at when the `.deb`s bumped this to `.so.4`, the change was requested to us during testing.